resources/views/faults/list.blade.php

```diff
@@ -0,0 +1,88 @@
+<div id="faults-table"></div>
+<script src="js/moment.min.js"></script>
+<script type="application/javascript">
+    $(document).ready(function () {
+        function status_translate(cell) {
+            if (typeof cell !== "string") {
+                cell = cell.getValue();
+            }
+            var translations = {
+                unseen: "@lang('faults.unseen')",
+                seen: "@lang('faults.seen')",
+                done: "@lang('faults.done')",
+                wont_fix: "@lang('faults.wont_fix')"
+            };
+            var colors = {
+                unseen: "text-info",
```

Comment:
Maybe text-secondary would look good here? Not sure.

Reply:
I agree, changed that.